app/assets/v2/js/grants/_new.js

```diff
@@ -296,6 +296,55 @@ if (document.getElementById('gc-new-grant')) {
           placeholder: 'Give a detailed desciription about your Grant'
         }
       };
+    },
+    computed: {
+      queryParams () {
+        let query = window.location.search.substr(1)
```

Comment:
a recommendation for future url params logic https://developer.mozilla.org/pt-BR/docs/Web/API/URLSearchParams